src/components/Table/features/RowPosition/utils.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  InitialTableState,
+  RowData,
+  RowModel,
+  Table,
+  TableState,
+} from "@tanstack/react-table";
+import { ACCESSOR_KEYS } from "../../../TableCreator/common/constants";
+import { DEFAULT_PAGINATION } from "../constants";
+
+/**
+ * Returns row model, with getter for row position.
+ * @param table - Table.
+ * @param getRowModel - Table getRowModel function.
+ * @returns row model.
+ */
+export function getRowModel<T extends RowData>(
+  table: Table<T>,
+  getRowModel: Table<T>[`getRowModel`]
+): RowModel<T> {
+  const rowModel = getRowModel();
+  rowModel.rows.forEach(({ id }, i) => {
+    rowModel.rowsById[id].getRowPosition = (): number =>
```

Comment:
Adding the function `getRowPosition` onto `rowModel.rowsById[id]` where `rowsById` is part of TanStack's rowModel API, which maps each `row` to its `id`.